Car/src/components/notifications/notifications-types/ApplicationAnswer.tsx

```diff
@@ -17,7 +17,13 @@ interface ApplicationAnswerProps {
     notificationTittle: string,
     notificationHeaderTittle: string,
     notificationHeaderMessage: string,
-    withWithdraw?: boolean
+    withWithdraw?: boolean,
+    IsDetailsTitleVisible: boolean,
+    IsDepartureTimeVisible: boolean,
+    IsFeeVisible: boolean,
+    IsAvailableSeatsVisible: boolean,
+    IsBaggageVisible: boolean,
+    IsStopsTitleVisible: boolean,
 }
```

Comment:
You can use '?' with boolean property not to mention them all every time(if the property is mentioned - its value is true, otherwise - it's false(undefined))